src/main/resources/com/mathworks/ci/RunMatlabCommandBuilder/help-matlabCommand.html

```diff
@@ -4,7 +4,10 @@
  <b>Run commands:</b> results = runtests(‘IncludingSubfolders’,true); assert(all(~[results.Failed]))<br>
  <b>Run a script:</b> runMyScript<br>
  <br>&nbsp;</br>
- <b>Recommendation:</b>If you require a number of MATLAB commands to execute your build, consider writing a MATLAB script and executing the script file instead.<br>
+ <b>Recommendation:</b><ul><li>If you require a number of MATLAB commands to execute your build, consider writing a MATLAB script and executing the script file instead.<br>
+ </li>
+ <li>Input text box supports commands only in ASCII formt. For locale specific commands, Write a MATLAB script using the supported chracater encoding and execute the script file.</li>
+ </ul>
  <b>Note:</b> The build will fail if the execution of any MATLAB command causes an error.
```

Comment:
@mw-hrastega request you to review these help text changes for locale-specific MATLAB command

Reply:
How about something like this?

The **Command** box supports only ASCII characters. If you use non-ASCII characters in your commands, write your commands to a .m file and invoke the script instead.

Reply:
Can we remove the existing recommendation? I find it kind of redundant now.

Reply:
Done.

![image](https://user-images.githubusercontent.com/47204011/86123635-7b819600-baf7-11ea-8bbd-275a7f18d8cc.png)

Reply:
Quick question. Can we also specify a .mlx file? If so, let's change ".m file" to "plain text file".

Reply:
I think we could run both , However instead of plain text we could use .m/.mlx ?

Reply:
Yes, that would be better. Lets replace ".m" with ".m or .mlx".

Reply:
changed it to .m or .mlx 


![image](https://user-images.githubusercontent.com/47204011/86142049-62391380-bb10-11ea-8b8c-53e8b75cb98a.png)